src/components/Definition/Definition.module.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../styles/variables";
+
+.definition_container {
+  padding: clamp(2px, 0.5vw, 5px) clamp(5px, 1vw, 10px);
+  text-align: center;
+  line-height: 1.5rem;
+  font-size: clamp(1rem, 2vw, 1.25rem);
+  background-color: $grey;
+  border-radius: 10px;
+
+  span {
```

Comment:
Not at all a concern in this case, more something that in another situation might make you look like a pro. When the casing is entirely visual, I add a `text-transform: uppercase;` to the CSS.

Reply:
I'm aware of the CSS, but I got lazy. haha. Will change.